python/activator/activator.py

```diff
@@ -74,13 +74,13 @@ def check_for_snap(
     instrument: str, group: int, snap: int, detector: int
 ) -> Optional[str]:
     prefix = f"{instrument}/{detector}/{group}/{snap}/{instrument}-{group}-{snap}-"
```

Comment:
Do you want to make use of the MDC feature that we use in butler logging? That lets you set an MDC record and after that every log message will get that content included without you having to explicitly add "prefix" to every log message.